RNA-seq/03-gastric_cancer_exploratory.Rmd

```diff
@@ -166,30 +128,41 @@ ggplot2::ggsave(pca_plot_file, plot = ggplot2::last_plot())
 We don't have batch information (i.e., when the samples were run) for this 
 particular experiment, but let's imagine that `SRR585574` and `SRR585576` were
 run separately from all other samples.
-We'll put this in new "toy" column in `sample_meta_df`.
+We'll add this as a new "toy" column in the sample data (`colData`) for `gene_summarized`.
+
+```{r extract-sample, live = TRUE}
+# Extract colData
+sample_info <- colData(gene_summarized)
+
+# Print out preview
+sample_info
+```
+
+Now we can add a new column with toy batch information and re-store the `colData()`. 
+
+```{r add-batch}
+# Add batch information
+sample_info$batch <- c("batch1", "batch1", "batch1", "batch1", "batch2", 
+                       "batch1", "batch2", "batch1")
 
-```{r batches}
-sample_meta_df$batch <- c("batch1", "batch1", "batch1", "batch1", "batch2", 
-                          "batch1", "batch2", "batch1")
+# Replace the sample information                        
+colData(gene_summarized) <- sample_info
```

Comment:
We aren't using this now, so we probably don't need to include this step: just adding it to `vst_data` (with a bit more explanation) should be fine.